docs/content/index.md

```diff
@@ -17,7 +17,7 @@ These environments can be extended, version controlled, and shared, so you can t
 
     Runs natively on ARM64 (Apple Silicon) and AMD64 machines.
 
-    * macOS Big Sur (11) or higher, [mostly](./users/usage/faq.md#can-i-run-ddev-on-an-older-mac)
+    * macOS Big Sur (11) or higher
```

Comment:
```suggestion
    * macOS Ventura (12) or higher. This is primarily driven by the available Docker providers.
```

Might as well update while we're here.  Homebrew is only the past two releases, so that limits colima and lima. Docker Desktop is only current release and previous two.